pyproject.toml

```diff
@@ -6,7 +6,6 @@ authors = [
     { name = "Aarni Koskela", email = "akx@iki.fi" },
 ]
 requires-python = ">=3.9"
-license = "MIT"
```

Comment:
Why was this removed, though? I'm pretty sure there are plenty of tools that read this field rather than classifiers?

Reply:
Because the classifier sets it on the license field in pypi iirc

Reply:
Ah, apparently it does the thing alright:

```
(ruuvitag-ble) ~/b/ruuvitag-ble (main) $ python
>>> import importlib.metadata as im
>>> im.metadata("ruuvitag-ble")
>>> m["License-Expression"]
'MIT'
>>>
```

Never mind! :)

Reply:
This PR should probably be reverted.`Hatchling` prefers the `license` field of classifier. PyPI will add support for `License-Expression` soon as well.

https://hatch.pypa.io/1.11/config/metadata/#spdx-expression
https://github.com/pypi/warehouse/issues/16620

Reply:
Opened #25